src/nbl/video/ILogicalDevice.cpp

```diff
@@ -6,6 +6,17 @@ using namespace nbl::video;
 
 E_API_TYPE ILogicalDevice::getAPIType() const { return m_physicalDevice->getAPIType(); }
 
+void ILogicalDevice::addJITIncludeLoader()
```

Comment:
could have just gone in the constructor next to where `m_compilerSet` is initialized

Reply:
PhysicalDevice forward declared

Reply:
move the constructor/creator to cpp file?